### DISCUSSION TITLE
Job Advertising

### DISCUSSION THREAD

**Discussion by CamBlack65** [community/community#39618] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi
We are looking to recruit a permanent software engineer in Sydney Australia. Do you have a section where I can advertise on your site?
Thanks

**Comment by BLKKKBVSIK** [comment]
Hello,

Github discussions are only for Github related discussions.
If you want to search for a profile to hire, making a post on the Github discussion forum is probably not the right place for that.

**Reply by CamBlack65** [reply]
I couldnt see where to ask the question, any ideas?
Thanks

**Comment by ossanna16** [comment]
Hi @CamBlack65, we're happy you're here 😀
However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!
